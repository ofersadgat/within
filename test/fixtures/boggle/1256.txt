btrsu
dhrdr
tsidt
ttnno
aietc