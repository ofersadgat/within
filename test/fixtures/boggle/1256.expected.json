{"word":"instinct","score":40}